### ISSUE TITLE
Rename `realm-management interfaces save`

### ISSUE THREAD

**Issue by bettio** [astarte-platform/astartectl#211]
`realm-management interfaces save` command might be renamed to something like `realm-management interfaces pull`  or `... fetch`, since `... save` might have an ambiguous semantic.

**Comment by matt-mazzucato** [comment]
I agree that the directionality of the command is not crystal clear. However, the counterpart of `save` is the `sync` command, which is not clear as well.

I think we can move to a more meaningful terminology,  deprecate the `save` and `sync` commands in favor of `pull` and `push` respectively.
